tests/unit/Makefile

```diff
@@ -60,15 +60,37 @@ $(VALGRIND_TESTS)::
 	@DYLD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$DYLD_LIBRARY_PATH" \
 	LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$LD_LIBRARY_PATH" \
 	S2N_VALGRIND=1 \
-	valgrind --leak-check=full --run-libc-freeres=no -q --error-exitcode=9 --gen-suppressions=all --log-fd=2 --num-callers=40 --leak-resolution=high --undef-value-errors=no --trace-children=yes --suppressions=valgrind.suppressions \
+	valgrind \
+		--leak-check=full \
+		--leak-resolution=high \
+		--trace-children=yes \
+		--run-libc-freeres=no \
+		-q --error-exitcode=123 \
+		--error-limit=no \
+		--num-callers=40 \
+		--undef-value-errors=no \
+		--log-fd=2 \
+		--suppressions=valgrind.suppressions \
 	./$(@:.valgrind=)
 
 $(PEDANTIC_VALGRIND_TESTS)::
 	@${CC} ${CFLAGS} -o $(@:.pedantic_valgrind=) $(@:.pedantic_valgrind=.c) ${LDFLAGS} 2>&1
 	@DYLD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$DYLD_LIBRARY_PATH" \
 	LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$LD_LIBRARY_PATH" \
 	S2N_VALGRIND=1 \
-	valgrind --leak-check=full --show-leak-kinds=all --errors-for-leak-kinds=all --run-libc-freeres=yes -q --error-exitcode=9 --gen-suppressions=all --num-callers=40 --leak-resolution=high --undef-value-errors=no --trace-children=yes --suppressions=valgrind.suppressions \
+	valgrind \
+		--leak-check=full \
+		--leak-resolution=high \
+		--trace-children=yes \
+		--run-libc-freeres=yes \
+		-q --error-exitcode=123 \
+		--error-limit=no \
+		--num-callers=40 \
```

Comment:
Should be bump this number while we are here?

Reply:
I would vote against bumping it. Their documentation mentions that having a high value can result in slower valgrind runtimes. And ideally it's just used for debugging rather than suppressions.